ArtU/ArtU/Views/SkillView.swift

```diff
@@ -6,43 +6,32 @@
 //
 
 // Mock Data
-var mockModules = ["Module 1", "Module 2", "Module 3", "Module 4"]
+struct Module: Identifiable, Hashable {
+    var id = UUID()
```

Comment:
Consider using let since id should be immutable

Reply:
The YouTube video I watched used `var` instead of `let`. Here is an article on why: https://developer.apple.com/forums/thread/651802.